scanner/config/config_test.go

```diff
@@ -123,6 +123,27 @@ func Test_MatcherConfig_validate(t *testing.T) {
 		err := c.validate()
 		assert.Error(t, err)
 	})
+	t.Run("when invalid indexer addr then error ", func(t *testing.T) {
+		for _, addr := range []string{"foo bar", "foo:bar", "80:80"} {
+			c := MatcherConfig{Enable: true, IndexerAddr: addr}
+			err := c.validate()
+			assert.Error(t, err)
+		}
+	})
+	t.Run("when valid addr then remote addr is set", func(t *testing.T) {
+		for _, addr := range []string{":8443", "localhost:443", "127.0.0.1:80"} {
```

Comment:
want to add another case for `scanner-v4-indexer.stackrox.svc:8443` as that is likely going to be what's used in prod?

Reply:
There is no need. The other cases cover the address in the above.